config/config-api/src/com/thoughtworks/go/domain/config/ConfigurationProperty.java

```diff
@@ -187,6 +187,10 @@ public ConfigErrors errors() {
         return configErrors;
     }
 
+    public boolean hasErrors(){
+        return !configErrors.isEmpty();
```

Comment:
The next set of commits use it, it is used in one of the (new) controllers to validate config properties of plugin tasks in an AJAXy manner.